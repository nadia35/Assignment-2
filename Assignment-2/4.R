printThirdElement = function(my_vec){
  
  if(length(my_vec) < 3){
    return("There are fewer than 3 elements in this vector")
  }
  else{
    return(my_vec[3])
  }
}
