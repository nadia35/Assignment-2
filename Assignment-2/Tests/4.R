source("4.R")

printThirdElement(1:5)
printThirdElement(1:100)
printThirdElement(100:1)
printThirdElement(1:2)
printThirdElement(99)