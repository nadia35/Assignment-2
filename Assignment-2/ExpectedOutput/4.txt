[1] 3
[1] 3
[1] 98
[1] "There are fewer than 3 elements in this vector."
[1] "There are fewer than 3 elements in this vector."
